app/views/user_case_infos/_form.html.erb

```diff
@@ -29,6 +29,11 @@
   <div class="field">
     <%= f.label :charge_type %><br>
```

Comment:
I know this wasn't part of your commit, but I'm pretty certain this `<br>` tags aren't necessary